src/common/components/header/index.tsx

```diff
@@ -14,6 +14,7 @@ const Header: React.FC = () => {
 	const [isMenuOpen, setIsMenuOpen] = useState(false);
 
 	const user = true;
+	const isModerator = true;
```

Comment:
Use from user property is_staff. I've already added in header

Reply:
yes of course This is temporary until I merge the authentication yet